### PR TITLE
Update LTPACrypto to support HW provider

### LINKED ISSUE #24955: PH53918: UnsupportedOperationException is thrown after upgrading to 22.0.0.10 or later
**Describe the bug**  

PH53918 has been assigned
A customer reported that after moving from 22.0.0.9 on z/OS to 22.0.0.12 both running on the same Java 11 level (11.0.17):
```
java.lang.UnsupportedOperationException: Hardware error, function getPrivateExponent has no meaning in hardware
at ibm.crypto.hdwrcca/com.ibm.crypto.hdwrCCA.provider.RSAPrivateHWKey.getPrivateExponent(RSAPrivateHWKey.java:86)
at com.ibm.ws.crypto.ltpakeyutil.LTPACrypto.rsaKey(LTPACrypto.java:1058)
at com.ibm.ws.crypto.ltpakeyutil.LTPADigSignature.generateRSAKeys(LTPADigSignature.java:49)
at com.ibm.ws.crypto.ltpakeyutil.LTPADigSignature.generateLTPAKeyPair(LTPADigSignature.java:76)
at com.ibm.ws.crypto.ltpakeyutil.LTPAKeyFileUtilityImpl.generateLTPAKeys(LTPAKeyFileUtilityImpl.java:50)
at com.ibm.ws.security.token.ltpa.internal.LTPAKeyFileCreatorImpl.createLTPAKeysFile(LTPAKeyFileCreatorImpl.java:62)
```

Between 22.0.0.9 and 22.0.0.12, the following code change is merged into the code area. 
https://github.com/OpenLiberty/open-liberty/pull/22340

**Steps to Reproduce**  

The customer had a hardware crypto provider at the top of the java.security file.  The workaround is to bring the OpenJCEPlus (or any JCE providers) above the hardware crypto provider. 
```
security.provider.1=IBMJCECCA                                      
security.provider.2=OpenJCEPlus                                    
security.provider.3=IBMZSecurity                                   
security.provider.4=SUN                                            
security.provider.5=SunRsaSign         
...
```

**Expected behavior**  
The error should not be thrown. .

**Diagnostic information:**  
 - OpenLiberty Version: 
 - Affected feature(s) appSecurity-2.0, appSecurity-3.0, appSecurity-4.0 
 - Java Version:  N/A
 - server.xml configuration (WITHOUT sensitive information like passwords)  NA

**Additional context**

### PR DESCRIPTION
**Pull request by una-tapa**
This PR is to fix https://github.com/OpenLiberty/open-liberty/issues/24955

### CODE REVIEW

**Comment by una-tapa** [comment]
#build

**Comment by una-tapa** [comment]
The build errors were irrelevant with this code change. Yet, let me try again to see if it clears up. 
 #build

**Comment by una-tapa** [comment]
Green build.  Sending review requests.

**Review by acdemyers — APPROVED** [review]

**Review by utle — COMMENTED** [review]
Can we put these code in a method? Also, is this code exist before?

**Comment by una-tapa** [comment]
@utle 
 "Does this code exist before?"
Yes, it did. Please refer to [the associated issue ](https://github.com/OpenLiberty/open-liberty/issues/24955 )for the history. 

"Can we put these code in a method?" 
We can make it into the method later.  Please refer to an internal MDV board issue 115 for the reason the refactoring is lower priority.
